dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/deprecated/tracker/TrackedEntityInstanceController.java

```diff
@@ -603,6 +608,8 @@ private void validateFileResource(FileResource fileResource, TrackedEntityAttrib
   private void setHttpResponse(HttpServletResponse response, FileResource fileResource) {
     response.setContentType(fileResource.getContentType());
     response.setContentLengthLong(fileResource.getContentLength());
-    response.setHeader(HttpHeaders.CONTENT_DISPOSITION, "filename=" + fileResource.getName());
+    response.setHeader(
+        HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=" + fileResource.getName());
```

Comment:
Not sure about adding `attachment` to images. Would be a regression at least.

Reply:
@JoakimSM It makes sense. I changed it back.

Reply:
Thanks, looks good now!